samples/nrf9160/gps/README.rst

```diff
@@ -0,0 +1,126 @@
+.. _gps_with_supl_support_sample:
+
+nRF9160: GPS sockets and SUPL client library
+############################################
+
+The GPS socket sample reads the `GPS`_ data from the `GNSS`_ socket and parses the received frames to interpret the frame contents.
+The GPS socket sample can also download the `A-GPS`_ data from a Secure User-Plane Location (SUPL) server using the SUPL client library, if the library is installed and enabled.
+See :ref:`supl_client` for information on enabling SUPL support for the sample.
+
+Overview
+********
+
+This sample operates in three different modes.
+Each of these modes depends on the state of the GPS module.
+
+Mode 1
+======
+
+This is the startup mode in which the sample waits for the GPS module to acquire a fix on the current position.
+Information on the processes is displayed on a console.
+
+Mode 2
+======
+
+The sample switches to the second mode when the GPS module has acquired at least one fix.
+In this mode the sample also displays information about the last good fix received from the GPS module.
```

Comment:
```suggestion
In this mode, the sample also displays information about the last good fix received from the GPS module.
```

Reply:
The overview of the modes was made much cleaner by Uma Prasseda, so this is an comment on the original commit.